src/aac/ui/menu_bar.py

```diff
@@ -0,0 +1,65 @@
+from tkinter import Menu, Tk
+
+
+def donothing():
+    print("Do nothing!")
+
+
+def menu_bar(root_window: Tk) -> Menu:
+    """
+    Creates and returns the AaC UI menu bar.
+
+    Args:
+        root_window (Tk): the window to attach the menu to
+
+    Returns:
+        The menu bar.
+    """
+    menu_bar = Menu(root_window, tearoff=0)
+    _add_file_menu(menu_bar)
+    _add_edit_menu(menu_bar)
+    _add_view_menu(menu_bar)
+    _add_help_menu(menu_bar)
+
+    return menu_bar
+
+
+def _add_file_menu(root_menu: Menu) -> None:
+    """Configures the file menu dropdown."""
+    file_menu = Menu(root_menu, tearoff=0)
+
+    file_menu.add_command(label="New File", command=donothing)
```

Comment:
Might be good to call it "New Model", or something instead of "New File" or have a menu that comes off of "New..." that expands to give "Data", "Enum", "Model", or "Use Case" options.

Reply:
I think this should be moved to a discussion.

Reply:
I'm fine with that.